### PR TITLE
Added PR for reviews

### PR DESCRIPTION
**Pull request by ani37**
for reviews

### CODE REVIEW

File: day2/exercise1/solution.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Println(string(data))
+}
+
+func main(){
+
+	str := []string{"quick", "brown", "fox", "lazy", "dog"}
```
Comment (Laghuvarapu) [inline day2/exercise1/solution.go]: change the name of the function

File: day2/exercise3/solution.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+		log.Fatal("lack of balance in account...")
+	}
+
+	m.Unlock()
+	wg.Done()
+}
+
+func main() {
+	var w sync.WaitGroup
+	var m sync.Mutex
```
Comment (Laghuvarapu) [inline day2/exercise3/solution.go]: make it as a global variable as we are using in above func also, then it becomes simple

File: day2/exercise2/solution.go [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+	var ratings = make(chan int, 200)
+	var average = make(chan int, 1)
+
+	go allocate(200, ratings)
+
+	var wg sync.WaitGroup
+	wg.Add(1)
+
+	go calculate(&wg, ratings, average)
```
Comment (Laghuvarapu) [inline day2/exercise2/solution.go]: change the name of the function

File: day2/exercise3/solution.go (hunk 2 above)
Comment (ani37) [reply day2/exercise3/solution.go]: done

File: day2/exercise2/solution.go (hunk 3 above)
Comment (ani37) [reply day2/exercise2/solution.go]: done

File: day2/exercise2/solution.go [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+func allocate(noOfStudents int, ratings chan int) {
+	for i := 0; i < noOfStudents; i++ {
+		time.Sleep(time.Duration(rand.Intn(100)) * time.Millisecond)
+		rating := rand.Intn(10) + 1
+		ratings <- rating
+	}
+	close(ratings)
+}
+
```
Comment (AnandPr123) [inline day2/exercise2/solution.go]: You have to take ratings from each classmate concurrently.

Comment (AnandPr123) [reply day2/exercise2/solution.go]: You are finding the ratings synchronously, although you are using goroutine.

Comment (ani37) [reply day2/exercise2/solution.go]: yes thanks, I have added the suggested changes.

**Pull request closed without merging**